.run-clang-format

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-find . -type f -regex '.*\.\(cpp\|hpp\|cc\|cxx\|h\|hxx\)' -exec clang-format-8 -style=file -i {} \;
```

Comment:
This is why btConvexHullComputer is disagreeing with CI. Install clang-format-8 and switch it back to the way it was and your local run will match CI.